pyproject.toml

```
@@ -5,5 +5,6 @@ build-backend = "setuptools.build_meta"
 [project]
 requires-python = ">= 3.10"
 name = "spycilab"
+dependencies = ["pyyaml >=  6.0.2"]
```

```suggestion
dependencies = ["pyyaml >=  5.4.1"]
```
On my ubuntu 22 machine this is the highest version I get through pip....